shared/constants/wallets.js

```diff
@@ -110,13 +110,13 @@ const paymentResultToPayment = (w: RPCTypes.PaymentOrErrorLocal) => {
     id: p.id,
     note: p.note,
     noteErr: p.noteErr,
-    source: p.source,
-    sourceType: partyTypeToString[p.sourceType],
+    source: p.fromUsername || p.fromAccountID,
```

Comment:
Should this line be:
```
     source: p.fromUsername || p.fromAccountID || p.fromAccountName,
```
?

Reply:
fromAccountID is always set. But it shouldn't be much like that, this was just enough to be a good starting point with valid flow.